packages/core/src/agent.ts

```diff
@@ -12,6 +12,10 @@ export interface AgentError extends BaseAgentMessage {
   body: string;
 }
 
+export interface AgentSuccess extends BaseAgentMessage {
```

Comment:
🤔 

I see the appeal of having a singular "success" response, but I wonder if we should future-proof this with a dedicatd `AgentReloadResponse` type, in anticipation of additional data coming eventually (i.e., what changed, how many channels restarted, etc)